src/index.html

```diff
@@ -81,7 +83,209 @@ <h3>I'm a Software Developer</h3>
         </div>
       </div>
     </section>
+
+    <!-- headline-container end -->
+
+    <!-- My Recent Works section start -->
+    <section class="section-works">
+      <article class="Recent-work">
+        <h2 class="works-title">My Recent Works</h2>
+        <img class="feature-image" src="./images/background-2.svg" alt="feature image about works">
+        <h3 class="title-post">Multi-Post Stories</h3>
+        <p class="post-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+          dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
```

Comment:
The grey background shown below does not seem to appear on the figma design. I think you should remove it in order to follow the design closely.

Your design:
![image](https://user-images.githubusercontent.com/20554583/213453754-f57369e3-dcb4-48a5-b2a6-7dc498ea4015.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/213454013-7f4e309f-8149-4082-8f0b-4b0d80a3cf99.png)

Reply:
resolved it